js/arithmetic-quiz.js

```diff
@@ -36,16 +39,23 @@ H5P.ArithmeticQuiz = (function ($) {
         incorrectText: 'Incorrect. Correct answer was :num',
         durationLabel: 'Duration in hours, minutes and seconds.',
         humanizedQuestion: 'What does :arithmetic equal?',
+        humanizedEquation: 'For the equation :equation, what does :item equal?',
+        humanizedVariable: 'What does :item equal?',
         plusOperator: 'plus',
         minusOperator: 'minus',
         multiplicationOperator: 'times',
         divisionOperator: 'divided by',
+        equalitySign: 'equal',
         slideOfTotal: 'Slide :num of :total'
       }
     }, options);
     self.currentWidth = 0;
 
-    self.gamePage = new H5P.ArithmeticQuiz.GamePage(self.options.arithmeticType, self.options.maxQuestions, self.options.UI, id);
+    // Reset equation type if not selected
+    if (self.options.quizType === H5P.ArithmeticQuiz.QuizType.ARITHMETIC) {
+      self.options.equationType = undefined;
+    }
+    self.gamePage = new H5P.ArithmeticQuiz.GamePage(self.options.arithmeticType, self.options.equationType, self.options.maxQuestions, self.options.useFractions, self.options.UI, id);
```

Comment:
Since this pull request is adding a new mode, I suggest the following:
- Create a separate question generator class per mode (that offers the same public "interface"/functions).
- Initialize the correct generator here, and dependency inject the instance into GamePage

Please let me know if this is unclear :)